src/components/DynamicFormContainer/DocumentPreview/DocumentPreview.tsx

```diff
@@ -5,19 +5,21 @@ import {
   renderDocument,
   FrameActions,
 } from "@govtechsg/decentralized-renderer-react-components";
-import { v2 } from "@govtechsg/open-attestation";
+import { v2, v3 } from "@govtechsg/open-attestation";
+import { isV3Document } from "../../../common/utils";
 
 type Dispatch = (action: HostActions) => void;
 
 interface DocumentPreview {
-  document: v2.OpenAttestationDocument;
+  document: v2.OpenAttestationDocument | v3.OpenAttestationDocument;
 }
 
 export const DocumentPreview: FunctionComponent<DocumentPreview> = ({ document }) => {
   const toFrame = useRef<Dispatch>();
   const [height, setHeight] = useState(0);
-
-  const rendererTemplate = document["$template"] as v2.TemplateObject;
+  const rendererTemplate = isV3Document(document)
+    ? document["openAttestationMetadata"]["template"]
+    : (document["$template"] as v2.TemplateObject);
```

Comment:
maybe can do a `getTemplate` utility at oa side, but anyway can it be dot notation format? 
```suggestion
  const rendererTemplate = isV3Document(document)
    ? document.openAttestationMetadata.template
    : (document.$template as v2.TemplateObject);
```

Reply:
removed, change to retrieve getTemplateURL in oa utils